src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+
+  background-color: #f1f1f1;
+}
+
+.header {
+  box-sizing: border-box;
+  padding: 0 40px;
+  display: flex;
+  justify-content: space-between;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header_large {
+  /* max-width: 1200px; */
+  margin-bottom: 62px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+
+  list-style-type: none;
+}
+
+.nav__item {
+  margin: 0 10px;
+}
+
+.nav__link {
+  display: block;
+  padding: 23px 0 25px;
+
+  position: relative;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: black;
+  text-decoration: none;
+}
+
+.nav__link:hover,
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+
+  position: absolute;
+  top: calc(100% - 4px);
+  left: 0;
+
+  width: 100%;
+  border-bottom: 4px solid #00acdc;
+  border-radius: 8px;
+}
+
+.header_small {
+  max-width: 1024px;
+}
```

Comment:
Check and remove all styles which relevant for small header

Reply:
Check comments and pay attention on difference in tests between reference and test for understanding which styles must be fixed
![Screenshot from 2023-03-02 12-35-50](https://user-images.githubusercontent.com/33029864/222407010-903933a7-f733-4aab-b89b-44d1a8a6a643.png)